app/views/browse/_replication-controller-actions.html

```diff
@@ -0,0 +1,47 @@
+<div ng-if="(('replicationControllers' | canIDoAny) || (!deploymentConfigName && !autoscalers.length && { group: 'extensions', resource: 'horizontalpodautoscalers' } | canI : 'create'))" class="pull-right dropdown">
+  <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+    Actions
+    <span class="caret"></span>
+  </button>
+  <a href=""
+     class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+     data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+  <ul class="dropdown-menu actions action-button">
+    <li ng-if="'replicationcontrollers' | canI : 'update'">
+      <a ng-href="project/{{project.metadata.name}}/attach-pvc?kind=ReplicationController&name={{deployment.metadata.name}}"
+         role="button">Attach Storage</a>
+    </li>
+    <li ng-if="!deploymentConfigName && ('replicationcontrollers' | canI : 'update')">
```

Comment:
so we are inconsistent in how we are handling these, for Set Resource Limits we completely hide the action on any RC that is from a DC, but for Edit Health Checks we enable the action if you are the active RC for a DC but send you over to editing the DC's health checks.